models/sdss_binary_results

```diff
@@ -0,0 +1,3 @@
+sdss_binary
+epoch=489-val_loss=1.565.ckpt
```

Comment:
It's interesting the loss was 3 times smaller than before. It'd be interesting to see some of the images or what the precision/recall graphs from tensorboard look like. Don't worry though if you didn't save these  files

Reply:
![image](https://user-images.githubusercontent.com/12701492/141202753-61b36aaf-3f7f-4242-aa92-f3377e78a24a.png)

Reply:
![image](https://user-images.githubusercontent.com/12701492/141203641-1e976fce-08a9-4bb0-884d-c51e7b6bc61e.png)

Reply:
That's interesting the accuracy definitely increased (it was hovering around 0.95 before). Not sure what caused it though

Reply:
I think we should look at the accuracy in real data, I'll get that working before this afternoon

Reply:
Thanks for posting the images!